argilla-frontend/translation/es.js

```diff
@@ -0,0 +1,271 @@
+    settings: "Mi configuración",
+    docs: "Documentación",
+    logout: "Cerrar sesión",
+  },
+  settings: {
+    title: "Configuración del dataset",
+    datasetInfo: "Información del dataset",
+    seeYourDataset: "Ver el dataset",
+    editFields: "Editar campos",
+    editQuestions: "Editar preguntas",
+    editMetadata: "Editar propiedades de metadatos",
+    editVectors: "Editar vectores",
+    deleteDataset: "Eliminar el dataset",
+    deleteWarning: "Ten cuidado, esta acción no es reversible",
+    deleteConfirmation: "Confirma la eliminación",
+    deleteConfirmationMessage:
+      "Estás a punto de eliminar: <strong> {datasetName} </strong> del espacio de trabajo <strong> {workspaceName} </strong>. Esta acción no puede deshacerse",
+    yesDelete: "Sí, eliminar",
+    write: "Escribir",
+    preview: "Vista previa",
+    uiPreview: "Vista previa de la interfaz de usuario",
+  },
+  button: {
+    ignore_and_continue: "Ignorar y continuar",
+    login: "Iniciar sesión",
+    "hf-login": "Iniciar sesión con Hugging Face",
+    sign_in_with_username: "Iniciar sesión con usuario",
+    cancel: "Cancelar",
+    continue: "Continuar",
+    delete: "Eliminar",
+    tooltip: {
+      copyToClipboard: "Copiar en el portapapeles",
+      copyNameToClipboard: "Copiar el nombre del dataset al portapapeles",
+      copyLinkToClipboard: "Copiar enlace del dataset al portapapeles",
+      goToDatasetSettings: "Configuración del dataset",
+      datasetSettings: "Configuración del dataset",
+    },
+  },
+  to_submit_complete_required:
+    "Para enviar completa \nlas respuestas requeridas",
+  some_records_failed_to_annotate: "Algunos registros no fueron anotados",
+  changes_no_submit: "No envió sus cambios",
+  bulkAnnotation: {
+    recordsSelected:
+      "1 registro seleccionado | {count} registros seleccionados",
+    recordsViewSettings: "Tamaño de registro",
+    fixedHeight: "Colapsar registros",
+    defaultHeight: "Expandir registros",
+    to_annotate_record_bulk_required: "No hay registro seleccionado",
+    select_to_annotate: "Seleccionar todo",
+    pageSize: "Tamaño de página",
+    selectAllResults: "Selecciona todos los registros coincidentes {total}",
+    haveSelectedRecords: "Has seleccionado todos los registros {Total}",
+    actionConfirmation: "Confirmación de acción en bloque",
+    actionConfirmationText:
+      "Esta acción afectará a {total} registros, ¿Quiere continuar?",
+    allRecordsAnnotated: "{total} registros han sido {action}",
+    affectedAll: {
+      submitted: "enviados",
+      discarded: "descartados",
+      draft: "guardados como borrador",
+    },
+  },
+  shortcuts: {
+    label: "Atajos",
+    pagination: {
+      go_to_previous_record: "Anterior (←)",
+      go_to_next_record: "Siguiente (→)",
+    },
+  },
+  questions_form: {
+    validate: "Validar",
+    clear: "Borrar",
+    reset: "Reiniciar",
+    discard: "Descartar",
+    submit: "Entregar",
+    draft: "Guardar como borrador",
```

Comment:
This text is too large for the button. New suggestion:
```suggestion
    draft: "Guardar borrador",
```